### PR TITLE
Update waze_travel_time.markdown

### PR DESCRIPTION
**Pull request by langestefan**
Text describing examples was cluttered to one line so I fixed that. Fixed one grammar error as well while I was at it.

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/waze_travel_time.markdown [hunk 1]
```diff
@@ -69,36 +69,37 @@ template:
...
 #### Tracking entity in Imperial Units
 
-Origin: person.paulus
-Destination: "725 5th Ave, New York, NY 10022, USA"
-Region: "US"
-Units: imperial
-Vehicle Type: motorcycle
+  - Name: "Somewhere in New York"
+  - Origin: `person.paulus`
+  - Destination: `725 5th Ave, New York, NY 10022, USA`
```
Comment (frenck) [inline source/_integrations/waze_travel_time.markdown]: We should not be using code fenced blocks for things that are not code.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/waze_travel_time.markdown (hunk 1 above)
Comment (langestefan) [reply source/_integrations/waze_travel_time.markdown]: Do you have a suggestion on what to use instead? It's supposed to be an input string. I can use quotes instead if that makes more sense.

Comment (frenck) [reply source/_integrations/waze_travel_time.markdown]: Yeah, just quotes will do.

**Review by frenck — APPROVED** [review]
Thanks, @langestefan 👍

**Pull request merged**